ATiwari2000/task4.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<head>
+  <title>Task 4</title>
+  <style>
+    section {
+      padding-top: 50px;
+      text-align: center;
+      margin-bottom: 500px;
+      margin-top: 50px;
+    }

```

Comment:
I suggest you to try height: 100vh next time, which means 100% vertical height.
It will allow you to set height according to screen sizes.